music21/beam.py

```diff
@@ -158,6 +158,8 @@ def __init__(self, type=None, direction=None, number=None):  # type is okay @Res
         # represents which beam line referred to
         # 8th, 16th, etc represented as 1, 2, ...
         self.number = number
+
+        # this should be called something else.
```

Comment:
I think that this id and the beam id below can be removed.  I've never used them.